docs/reference/config.md

```diff
@@ -72,7 +80,32 @@ sql:
       emit_async_querier: true
       query_parameter_limit: 5
 ```
-  
+
+### database
+
+The `database` mapping supports the following keys:
+
+- `url`:
```

Comment:
I know this isn't the ideal time to offer naming suggestions, but we can't really change this after release so it's worth a brief discussion. `uri` is better than `url` in this context I think, if we want to indicate that we are only supporting the URI-style connection string format for various drivers (and not other formats e.g. key/value pairs).

I took a quick read of the [postgres](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) and [mysql](https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html) and [sqlite](https://www.sqlite.org/c3ref/open.html) docs on this and they all mention "URI."